### ISSUE TITLE
Not up to date with astropy v5.1

### ISSUE THREAD

**Issue by nkphysics** [nkphysics/autoNICER#13]
There are unit warnings coming up when using astropy v5.1. This indicates the use of deprecated functionality in astropy and as such autonicer should be brought up to date to work with astropy v5.1.

A **temporary** fix was applied with v1.0.2 of autonicer, but a more permanent fix should be accounted for in future versions.

**Comment by nkphysics** [comment]
Issue comes up when parsing astroquery.heasarc query after initial input of settings.
```
WARNING: UnitsWarning: 'DEGREE' did not parse as fits unit: At col 0, Unit 'DEGREE' not supported by the FITS standard.  If this is meant to be a custom unit, define it with 'u.def_unit'. To have it recognized inside a file reader or other code, enable it with 'u.add_enabled_units'. For details, see https://docs.astropy.org/en/latest/units/combining_and_defining.html [astropy.units.core]

WARNING: UnitsWarning: 'MJD' did not parse as fits unit: At col 0, Unit 'MJD' not supported by the FITS standard. Did you mean MJ or mJ? If this is meant to be a custom unit, define it with 'u.def_unit'. To have it recognized inside a file reader or other code, enable it with 'u.add_enabled_units'. For details, see https://docs.astropy.org/en/latest/units/combining_and_defining.html [astropy.units.core]
```

**Comment by nkphysics** [comment]
This seems to be a known issue on astroquery's end.
See the following issue and pull requests via astroquery for further details.

Issue: [https://github.com/astropy/astroquery/issues/2333](url)
Pull Requests: 
[https://github.com/astropy/astroquery/pull/2349](url)
[https://github.com/astropy/astroquery/pull/2353](url)

**Comment by nkphysics** [comment]
Resolved with #27
